src/priorities/priorities_overlords.ts

```diff
@@ -22,7 +22,8 @@ export let OverlordPriority = {
 		destroy         : 300,
 		healPoint       : 301,
 		siege           : 302,
-		controllerAttack: 399
+		controllerAttack: 399,
+		harass			: 571,
```

Comment:
all offense priorities must be 300-399. make a separate category if you need a different priority

Reply:
Harass is meant to be a very low priority drain assault that occasionally spawns, so I figured a lower priority was fitting.